packages/web-components/src/search/index.ts

```diff
@@ -0,0 +1,67 @@
+import { attr, DOM } from '@microsoft/fast-element';
+import { Search as FoundationSearch, SearchOptions, searchTemplate as template } from '@microsoft/fast-foundation';
+import { fillColor, neutralFillInputRecipe } from '../design-tokens';
+import { Swatch } from '../index';
+import { searchStyles as styles } from './search.styles';
+
+/**
+ * Search appearances
+ * @public
+ */
+export type SearchAppearance = 'filled' | 'outline';
+
+/**
+ * The Fluent search class
+ * @internal
+ */
+export class Search extends FoundationSearch {
+  /**
+   * The appearance of the element.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: appearance
+   */
+  @attr
+  public appearance: SearchAppearance = 'outline';
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    DOM.queueUpdate(() => {
+      fillColor.setValueFor(
```

Comment:
Yeah, I'd like some clarity on what is being done here - are we doing this as a work around to an issue? If so, we should halt and solve the issue itself ASAP. Setting universal context here to the focus state is going to likely create WCAG issues around color contrast. We should always "do the right thing" rather than "make it look right".

Reply:
The issue I was referencing is described in [FAST 5286](https://github.com/microsoft/fast/issues/5286). I'd love to solve that color problem, but I don't think it will be quick. I think Nick's proposal works, but it means we need to retrofit the styles to update the `fillColor` based on state changes and recipes rather than to set the css `background-color` because that gives the recipes no way to respond. This shows up most with stealth because it's most noticeable when it has a backplate and shouldn't. It also causes problems with hint text because that's right at 4.5:1 contrast.

Reply:
I think I have a larger concern around using Button here in the Foundation template which I've started an issue to discuss: https://github.com/microsoft/fast/issues/5451

It doesn't change the context problem but would move this to be similar in implementation to the tree item expand/collapse most like (which I think makes more sense for the clear button in this scenario)